### PR TITLE
this commit fixes the issue #811.

### PR DESCRIPTION
**Pull request by vahdat-ab**
I'm trying to see if the windows build will fail again or not.
It passes in m y machine.

### CODE REVIEW

**Comment by vahdat-ab** [comment]
@Nava2 I wonder if you can tell me what's the reason for this failure on appveyor? How I can force the appveyor to do it again. it workds on my Windows. The only failure I have is the old one which is `BuilderTest`

**Comment by Nava2** [comment]
Could you try changing this [line](https://github.com/umple/umple/blob/issue811/appveyor.yml#L16) to not redirect the output, i.e. just `cinst ant`? 

By doing this, we can see why the command fails. The Ruby update doesn't fail, its the chocolatey install.

File: cruise.umple/src/UmpleInternalParser_CodeParserHandlers.ump [hunk 1]
```diff
@@ -33,15 +33,30 @@ class UseStatementParserAction{
...
+    if (data!=null && data.getFullFileAddress()!=null)    index = data.getFullFileAddress().lastIndexOf("/") ;
+    if (index==-1) index = data.getFullFileAddress().lastIndexOf("\\");  
+    if (index!=-1){
+    	path = data.getFullFileAddress().substring(0,index+1 );
+    }
+    
     synchronized(data.getHasParsed())
     {
-      if(!data.getHasParsed().contains(value))
+        fileName = data.getAnalyzer().getFile().getAbsoluteFile().getParentFile().getAbsolutePath() + File.separator + value;
```
Comment (Nava2) [inline cruise.umple/src/UmpleInternalParser_CodeParserHandlers.ump]: Completely aside, please fix the indentation here, it's confusing. I can't tell what is part of what `if` clause.

**Comment by Nava2** [comment]
I think the chocolatey install is failing, I have no idea why. The package hasn't been updated, and it fails locally.

I just ran the command, `cinst ant` locally, and it failed due to a server problem:

```
Attempt to get headers for http://archive.apache.org/dist/ant/binaries/apache-ant-1.9.6-bin.zip failed.
   Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (503) Server Unavailable."
 Downloading ant 32 bit
   from 'http://archive.apache.org/dist/ant/binaries/apache-ant-1.9.6-bin.zip'
 Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (503) Server Unavailable."
 At C:\ProgramData\chocolatey\helpers\functions\Get-WebFile.ps1:81 char:3
 +   $res = $req.GetResponse();
 +   ~~~~~~~~~~~~~~~~~~~~~~~~~
     + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
     + FullyQualifiedErrorId : WebException
 You cannot call a method on a null-valued expression.
 At C:\ProgramData\chocolatey\helpers\functions\Get-WebFile.ps1:166 char:3
 +   $res.Close();
 +   ~~~~~~~~~~~~
     + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
     + FullyQualifiedErrorId : InvokeMethodOnNull
 Hashes match.
```

I'm thinking this is a problem with the apache servers. This would agree with my sentiment.

Solved why it happens, I think, now whether to accept the PRs is up to you, @vahdat-ab and @TimLethbridge.

**Comment by TimLethbridge** [comment]
I am rerunning the appveyor builds to see if the Apache servers have come back up.

**Comment by TimLethbridge** [comment]
appveyor is now passing on other PRs so I suspect the problem will be solved when these are rerun, as is happening

**Comment by TimLethbridge** [comment]
@vahdat-ab  - the appveyor problem has resolved for this and the other PRs

**Pull request merged**